src/utilities/jobs/emails/payments/send-payment-requested-notice.ts

```diff
@@ -29,7 +29,7 @@ const sendPaymentRequestedNotice = async ({
   const payerUserId = paymentData.payer_user_id;
   const payeeUserId = paymentData.payee_user_id;
   const amountOwed = paymentData.amount;
-  const cancelToken = paymentData.cancel_token;
+  const cancel_token = paymentData.cancel_token;
```

Comment:
for frontend stuff I want it to remain camelCase

Reply:
change added